### PR TITLE
Commenting out lines to fix the bug of no such file or directory

### LINKED ISSUE #2989: bug: Mage Install runs into "no such file or directory"
[It's somewhere around here](https://github.com/getporter/porter/blob/main/magefile.go#L656)

```
mage Install
installing Porter from bin to /Users/schristoff/.porter
Error: remove /Users/schristoff/.porter/mixins/arm/arm: no such file or directory
```

### PR DESCRIPTION
**Pull request by Bharadwajshivam28**
# What does this change
This fixes the Bug when we do Mage Install an error comes "no such file or directory"

# What issue does it fix
fixes #2989

# Notes for the reviewer
Please look into some points i raised in the issue comment section

# Checklist
- [ ] Did you write tests?
- [ ] Did you write documentation?
- [ ] Did you change porter.yaml or a storage document record? Update the corresponding schema file.
- [ ] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️
[contributors]: https://getporter.org/src/CONTRIBUTORS.md

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -679,7 +679,7 @@ func Install() {
 		// Removing the file first clears the cache so that we don't run into "zsh: killed MIXIN..."
 		// See https://stackoverflow.com/questions/67378106/mac-m1-cping-binary-over-another-results-in-crash
 		// See https://openradar.appspot.com/FB8914231
-		mgx.Must(os.Remove(filepath.Join(destDir, mixin+xplat.FileExt())))
+		//mgx.Must(os.Remove(filepath.Join(destDir, mixin+xplat.FileExt())))
```
Comment (schristoff) [inline magefile.go:682]: This line was there for an issue on M1 Macs where there would be an error about the executable not being signed properly and wouldn't execute. Are you able to confirm you're not able to reproduce that error?
If so, you can just delete this line and all the comments addressing it :)

Comment (Bharadwajshivam28) [reply magefile.go:682]: Hey @schristoff  give me some time will confirm it and drop a message here

**Comment by Bharadwajshivam28** [comment]
Hey @schristoff  I tested by removing line 682-683 . It worked fine.. so we can remove both the lines

**Comment by kichristensen** [comment]
I hit a similar issue and the same today. Almost the same eror can happen on line 648 and 649, when Porter is not installed initially

**Comment by schristoff** [comment]
Hey @kichristensen thanks for letting us know, mind if I ask, are you on Mac and if so what architecture?

**Comment by kichristensen** [comment]
@schristoff Unfortunately I'm not on Mac

**Comment by Bharadwajshivam28** [comment]
> @schristoff Unfortunately I'm not on Mac

I faced the same issue on Linux(Manjaro) while setting it up... but commenting those lines fixes it

**Pull request closed without merging**